src/test/java/org/itsallcode/openfasttrace/importer/legacytag/config/DescribedPathMatcherTest.java

```diff
@@ -0,0 +1,145 @@
+    }
+
+    @Test
+    public void testListBasedMatcherMatchesRelativePath()
+    {
+        createListMatcher("rel/path");
+        assertMatches("rel/path", true);
+        assertMatches("/rel/path", false);
+        assertMatches("rel/path/", true);
+        assertMatches("rel/path1", false);
+        assertMatches("rel/path.txt", false);
+        assertMatches("path", false);
+        assertMatches("rel", false);
+    }
+
+    @Test
+    public void testListBasedMatcherIsCaseInsensitiveUnderWindows()
+    {
+        OsDetector.assumeRunningOnWindows();
+        createListMatcher("rel/path");
+        assertMatches("REL/path", true);
+        assertMatches("rel/PATH", true);
+    }
+
+    @Test
+    public void testListBasedMatcherIsCaseSensitiveUnderUnix()
+    {
+        OsDetector.assumeRunningOnUnix();
+        createListMatcher("rel/path");
+        assertMatches("REL/path", false);
+        assertMatches("rel/PATH", false);
+    }
+
+    @Test
+    public void testListBasedMatcherMatchesAbsolutePath()
+    {
+        createListMatcher("/abs/path");
+        assertMatches("/abs/path", true);
+        assertMatches("abs/path", false);
+        assertMatches("/abs/path/", true);
+    }
+
+    @Test
+    public void testListBasedMatcherMatchesMultiplePaths()
+    {
+        createListMatcher("path1", "path2");
+        assertMatches("path1", true);
+        assertMatches("path2", true);
+        assertMatches("path2/", true);
+        assertMatches("/path2", false);
+        assertMatches("path", false);
+    }
+
+    @Test
+    public void testListBasedMatcherDescription()
+    {
+        createListMatcher("path1", "path2");
+        assertDescription("[path1, path2]");
+    }
+
+    @Test
+    public void testPatternMatcherDescription()
+    {
+        createPatternMatcher("path");
+        assertDescription("glob:path");
+    }
+
+    @Test
+    public void testPatternMatcherMatchesGlob()
+    {
+        createPatternMatcher("glob:path/**");
+        assertMatches("path", false);
+        assertMatches("path/", false);
+        assertMatches("path/file", true);
+        assertMatches("path/FILE", true);
+        assertMatches("path/dir/", true);
+        assertMatches("path/dir/file", true);
+    }
+
+    private void createListMatcher(final String... paths)
+    {
+        final List<Path> pathList = Arrays.stream(paths).map(Paths::get).collect(toList());
+        this.matcher = DescribedPathMatcher.createPathListMatcher(pathList);
+    }
+
+    private void createPatternMatcher(final String pattern)
+    {
+        this.matcher = DescribedPathMatcher.createPatternMatcher(pattern);
+    }
+
+    private void assertMatches(final String path, final boolean expected)
+    {
+        assertThat("path " + path, this.matcher.matches(Paths.get(path)), equalTo(expected));
+    }
+
+    private void assertDescription(final String expected)
+    {
+        assertThat(this.matcher.getDescription(), equalTo(expected));
+    }
+}
```

Comment:
Add tests for Windows-style paths too. Just to be safe.

Reply:
:heavy_check_mark: